### ISSUE TITLE
[Term Entry] Neural Networks ai adam optimization

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4528]
### Reason/inspiration (optional)

"We would like a new term entry in the `AI` concept for [neural-network](https://www.codecademy.com/resources/docs/ai/neural-networks): Adam Optimization. The entry should go in a new file under `docs/content/ai/concepts/neural-networks/terms/adam-optimization/adam-optimization.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use concept for stochastic gradient descent.
- Should contain information for this concept and syntax.
- Should have a Python codebyte with a working code snippet showcasing this concept.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by TimLyons87** [comment]
@codecademy-docs  I would like to be considered for this issue. It will be my first issue for the learn git and GitHub course. Thank you for your consideration.

**Comment by mamtawardhani** [comment]
Hey @TimLyons87! You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

**Comment by TimLyons87** [comment]
@mamtawardhani  Thank you very much for this issue assignment. Yes, this is my first assignment and I found out about it through the Learn Git and GitHub course. I have some machine learning experience that feel would makes this issue project a good fit. 

I plan to start on right away. Thanks again!

**Comment by mamtawardhani** [comment]
Unassigned the author due to inactivity

**Comment by pratik305** [comment]
@mamtawardhani  i would love to work on this

**Comment by mamtawardhani** [comment]
Hey @pratik305 You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.
